internal/venice-common/src/main/java/com/linkedin/venice/utils/HelixUtils.java

```diff
@@ -335,6 +336,12 @@ public static boolean isLiveInstance(String clusterName, String instanceId, Safe
     }
   }
 
+  public static MaintenanceSignal getClusterMaintenanceSignal(String clusterName, SafeHelixManager manager) {
```

Comment:
Could you please add a JavaDoc to explain how to interpret the return values? It appears that null means the cluster is not in MM.